### PR TITLE
fix #74617 re-library.com

### LINKED ISSUE #74617: re-library.com
### Issue URL (Anti Adblock Script)
 [https://re-library.com/translations/hero-king/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fre-library.com%2Ftranslations%2Fhero-king%2F)
### Comment
Username: `@TommyKing`
> 
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> https://re-library.com/2021/02/11/heros-daughter-chapter-176/
> 
> Custom userscripts:
> https://userscripts.adtidy.org/beta/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/beta/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 16
Browser: | PaleMoon 29.0.1
AdGuard driver: | TDI
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> AdBlockID
Other extensions used: | Eset NOD32

### PR DESCRIPTION
**Pull request by wout-codes**
#74617

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -434,7 +434,7 @@ tubsexer.com,homegrownfreaks.net,pornfd.com,bobs-tube.com,pornsexer.com#@#.ads-i
 !#endif
 !
 ! ai_front anti-adblock
-elektrikmen.com,world4.eu,doctorrouter.ru,gcertificationcourse.com,sagliktais.com,1ahaber.com,scubidu.eu,politics-dz.com,game-owl.com,freebulksmsonline.com,fantasyfootballgeek.co.uk,4allprograms.me,unityassets4free.com,hblinks.pro,hdhub4u.cc,exbulletin.com,apkmaza.net,janusnotes.com,tekfame.com,hunter-xhunter.com,sysnettechsolutions.com,wristreview.com,freewebcart.com,parzibyte.me,ftuapps.dev,sznlink.xyz,hinjakuhonyaku.com,toppremiumpro.com,mashtips.com,dramahd.me,gaypornmasters.com#%#//scriptlet("abort-on-property-write", "ai_front")
+re-library.com,elektrikmen.com,world4.eu,doctorrouter.ru,gcertificationcourse.com,sagliktais.com,1ahaber.com,scubidu.eu,politics-dz.com,game-owl.com,freebulksmsonline.com,fantasyfootballgeek.co.uk,4allprograms.me,unityassets4free.com,hblinks.pro,hdhub4u.cc,exbulletin.com,apkmaza.net,janusnotes.com,tekfame.com,hunter-xhunter.com,sysnettechsolutions.com,wristreview.com,freewebcart.com,parzibyte.me,ftuapps.dev,sznlink.xyz,hinjakuhonyaku.com,toppremiumpro.com,mashtips.com,dramahd.me,gaypornmasters.com#%#//scriptlet("abort-on-property-write", "ai_front")
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/antiadblock.txt:437]: The popup can be closed so please move this domain to the similar rule in annoyances

File: EnglishFilter/sections/content_blocker.txt [hunk 2]
```diff
@@ -1337,6 +1337,10 @@ gcertificationcourse.com###sticky-ad
 @@||contextual.media.net/dmedianet.js$domain=politics-dz.com
 @@||politics-dz.com/wp-content/plugins/ad-inserter/*/*.js
 @@||politics-dz.com/wp-content/plugins/ad-inserter/images/ads.png
+! https://github.com/AdguardTeam/AdguardFilters/issues/74617
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/content_blocker.txt:1340]: Antiadblock still triggers with these rules in user rules

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/content_blocker.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/content_blocker.txt:1340]: @Sergey-Lyapin Suggest me new rules for this. I have no idea right now.

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:437]: Please give me suggestions, I am not that good in writing CSS rules.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt]: Fixed.

File: EnglishFilter/sections/content_blocker.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/content_blocker.txt]: Fixed.

**Pull request merged**